### PR TITLE
Use Cookie::Baker to bake cookies in Plack::Response.

### PR DESCRIPTION
**Pull request by oalders**
Let me know if I've missed anything.  The test coverage already seemed pretty god.

### CODE REVIEW

**Comment by miyagawa** [comment]
@kazeburo thoughts? I'm surprised this hasn't been done yet?

**Comment by oalders** [comment]
I should have mentioned that this closes #478.  I apologize for the delay. :)

File: lib/Plack/Response.pm [hunk 1]
```diff
@@ -123,47 +124,25 @@ sub _body {
...
-    if ($expires =~ /^\d+$/) {
-        # all numbers -> epoch date
-        # (cookies use '-' as date separator, HTTP uses ' ')
-        my($sec, $min, $hour, $mday, $mon, $year, $wday) = gmtime($expires);
-        $year += 1900;
-
-        return sprintf("%s, %02d-%s-%04d %02d:%02d:%02d GMT",
-                       $WDAY[$wday], $mday, $MON[$mon], $year, $hour, $min, $sec);
+        my %cookie
+            = map { $_ => $val->{$_} } grep { $val->{$_} } @cookie_args;
```
Comment (kazeburo) [inline lib/Plack/Response.pm]: This copy is not required.

`Plack::Response->cookies` allows scalar values.
see https://metacpan.org/pod/Plack::Response#cookies

```
$res->cookies->{foo} = 123;
$res->cookies->{foo} = { value => '123' };
```

Comment (oalders) [inline lib/Plack/Response.pm]: Good catch! I've force-pushed an updated commit.

**Pull request merged**